File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
import java.util.*;

class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String id = id_pw[0];
        String pw = id_pw[1];
        for (String[] user : db) {
            if (user[0].equals(id)) {
                return user[1].equals(pw) ? "login" : "wrong pw";
            }
        }
        return "fail";
    }
}

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.js
function solution(id_pw, db) {
    const [id, pw] = id_pw;
    for (let [dbId, dbPw] of db) {
        if (dbId === id) {
            return dbPw === pw ? "login" : "wrong pw";
        }
    }
    return "fail";
}
